calebwagner/frontend-capstone
# create database

#### Changes Made
1. Modified file NotebookProvider.js to include add, delete, edit, and getter function.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-create-notebook-db
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
    > check file for added functions
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
